README.md

```diff
@@ -224,3 +232,11 @@ status:
       apiGroup: rbac.authorization.k8s.io
       ...
 ```
+
+## Ulterior motives
+
+Our "hidden" agenda with `combo` is for it to:
+- serve as an example of best-practices when building operators
+- be used to dog food operator-framework packaging formats (e.g. OLM, rukpak, etc)
+- become a dependency of operators that require post-install configuration (e.g. RBAC scoping, secret creation, etc)
+- serve as a kruft-free way to up-skill new maintainers through easy contributions
```

Comment:
nit: Let's use the common spelling of "cruft"

Reply:
Good call 😅